tests/common/other.rs

```diff
@@ -1,6 +1,6 @@
 use anyhow::{Context, Result};
 use flate2::read::GzDecoder;
-use maturin::BuildOptions;
+use maturin::{BuildOptions, Metadata21};
```

Comment:
```suggestion
use maturin::BuildOptions;
```

I believe `Metadata21` is unused.